modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -120,22 +122,24 @@ If you don't see the *Connectors* tab, verify that you have a Titanium subscript
 | *Performance* a|
 
 * Average response time inbound
-* Average Response Time Grouped by Endpoint Inbound
+* Average response time grouped by endpoint - Inbound
 * Mule messages
 * Response time graphs: 99, 90, 75, and 50 percentile inbound
 * Average response time outbound
-* Average response time grouped by endpoint outbound
-* Response time graphs: 99, 90, 75, and 50 percentile outbound
+* Average response time grouped by endpoint - Outbound
+* Response time graphs: 99, 90, 75, and 50 percentile - Outbound
 //* Response Time graphs: 99, 90, 75, and 50 Percentile Inbound
 //* Average Response Time: Outbound, Grouped by Endpoint Outbound
 | *Failures* a|
 
 * Total failed inbound requests
 * Total failed outbound requests
-* All inbound grouped by response type
-* All outbound grouped by response type
-* Total failed inbound grouped by endpoint
-* Total failed outbound grouped by endpoint
+* All inbound requests grouped by response type
+* All outbound requests grouped by response type
+* Failed inbound requests: Response time by endpoint
+* Total failed inbound requests grouped by endpoint
+* Failed outbound requests: Response time by HTTP endpoint (only HTTP endpoints are shown)
+* Total failed outbound requests grouped by HTTP endpoint (only HTTP endpoints are shown)
 | *JVM* a|
 
```

Comment:
HIDDEN, UNEDITED

* Par new collection count, Par new collection time
* Classes: Loaded, Loaded total, Unloaded
* Heap: Committed, Used
* Thread Count: Number of simultaneous requests
* JVM Uptime: Uptime of the JVM
* Par Eden graphs (Used, Max, Init, and Committed): Usage metrics for the Par Eden Space pool
* Par Survivor graphs (Used, Max, Init, and Committed): Usage metrics for the Par Eden Space pool
* Metaspace graphs (Used, Max, Init, Committed): Usage metrics for Java memory space
* Code Cache graphs (Used, Max, Init, Committed): Usage metrics for the Code Cache pool
* Compressed Class Space graphs (Used, Max, Init, Committed): Usage metrics for values of the metadata for compressed class pointers
* Committed Virtual Memory: Amount of virtual memory guaranteed to be available to the running process
| *Infrastructure* a|
* Application Process CPU Load: Amount of CPU load used by the JVM
* Application Process CPU Minutes: CPU time used by the process on which the JVM is running
* Operating System Swap Space: Amount of memory the system is swapping
* CPU Utilization %: Percentage of CPU used by each worker over time
* Memory Utilization: Amount of memory used by each worker over time
* Total System Processors: Number of system processors for workers available over time
* Total System Memory: Amount of system memory available for workers over time. +
If you have a Titanium subscription and the selected application is hybrid, this graph changes to *System Memory*.
* System Memory: This graph displays only if you have a Titanium subscription and the selected application is hybrid. This graph displays the total free system memory per worker.
* Thread Count - Server: Number of simultaneous requests for a worker over time
.Example: Infrastructure Dashboard
image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
| *Custom metrics* a|
The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
|===
== See Also
* xref:dashboard-custom-config.adoc[Configuring Custom Dashboards and Charts]
* xref:reports.adoc[Reports]
* xref:save-builtin-dashboard-as-custom.adoc[Save a Built-in Dashboard as a Custom Dashboard]